host/crypto/windows/rsa.c

```diff
@@ -0,0 +1,556 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "../rsa.h"
+#define WIN32_NO_STATUS
```

Comment:
If I don't have the `WIN32_NO_STATUS` macro, then there's a lot of macro redef warnings no matter how I rearrange those headers. The redefs have due to ntstatus.h and um/winnt.h containing the same defs. You still need ntstatus.h though since STATUS_SUCCESS isn't defined.